roles/ci_lvms_storage/templates/lvms-cluster.yaml.j2

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: lvmcluster
```

Comment:
I've deployed  >1 LVMS cluster on the same OpenShift by calling this role >1 time and overriding the `cifmw_lvms_namespace` each time. However, I didn't test them both at the same time. In that scenario you'd want to override the `cifmw_lvms_disk_list` each time too and it would make sense to override the name `lvmcluster` so I could make that a variable.

Reply:
fixed

https://github.com/openstack-k8s-operators/ci-framework/compare/ce29d5f1c119885a3f605f70d7fc36737417ef97..3054c17e7a4411b885ed79c4948755d56ebf0589